docs/articles/commands/processors/slash_commands/missing_commands.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: articles.commands.command_processors.slash_commands.missing_commands
+title: Missing Commands
+---
+
+# Missing Commands
+
+> Help! I registered all my slash commands but they aren't showing up!
+
+When the Discord App (the client you use, not the bot) starts up, it fetches all the commands that are registered with the bot and caches them to a Discord channel. Unfortunately this cache is not properly updated in real-time, so it may take a while for the commands to show up.

```

Comment:
The fetching happens when the client enters a certain context for the first time. A context is channel scoped. Also the cache is per context